src/site/content/en/fast/use-lighthouse-for-performance-budgets/index.md

```diff
@@ -0,0 +1,65 @@
+# Use Lighthouse for Performance Budgets
+
+Lighthouse now supports performance budgets. This feature, LightWallet, can be set up in under five minutes and provides feedback on the size and quantity of page resources.
+
+## Install Lighthouse
+
+LightWallet is available in the comand line version Lighthouse (versions 5 & up). 
+
+Install Lighthouse:
+
+```bash
+$ npm install -g lighthouse
+```
+
+## Create a Budget
+
+Create a file named `budget.json`. In this file add the following JSON:
+
+```json
+[
+    {
+        "resourceSizes": [
+            {
+                "resourceType": "script",
+                "budget": 125
+            },
+            {
+                "resourceType": "total",
+                "budget": 300
+            }
+        ],
+        "resourceCounts": [
+            {
+                "resourceType": "third-party",
+                "budget": 10
+            }
+        ]
+    }
+]
+```
+
+This example budget.json file sets three separate budgets:
+
+*   A budget of 125 KB for the total amount of JavaScript on the page.
+*   A budget of 300 KB for the overall size of the page.
+*   A budget of 10 requests for the numer of requests made to third-party origins.
+
+A `budget.json` file can contain any number of budgets. These are the resource types supported by LightWallet: `document`, `font`, `image`, `media`, `other`,`script`, `stylesheet`, `third-party`, and `total`.
+
+
+## Run Lighthouse
+
+Run Lighthouse using the `--budget-path` flag. This flag tells Lighthouse the location of your budget file. (Note: A budget file does not have to be named `budget.json`.)  
+
+```bash
+$ lighthouse https://example.com --budget-path=./budget.json
+```
+
+## View Results
+
+If LightWallet has been configured correctly, the Lighthouse report will contain a “Budgets” section within the “Performance” category.
+
+<img src="lightwallet.png" class="w-screenshot" alt="'Budgets' section of the Lighthouse report">
```

Comment:
I'm not seeing this image. It should probably be ./lightwallet.png.

Also, you should use proper image formatting.

Looks something like this:

<figure class="w-figure">
  <img class="w-screenshot w-screenshot--filled" src="./lightwallet.png" alt="A LightWallet report showing which assets are over the file size budget." loading="lazy">
</figure>